src/common/misc/LazySearchBar.ts

```diff
@@ -2,16 +2,26 @@ import m, { Children, ClassComponent, Vnode } from "mithril"
 import type { SearchBar, SearchBarAttrs } from "../../mail-app/search/SearchBar.js"
 import { LazyLoaded } from "@tutao/tutanota-utils"
 import { client } from "./ClientDetector.js"
+import { CalendarSearchBar } from "../../calendar-app/calendar/search/CalendarSearchBar.js"
```

Comment:
common should not be importing things from mail-app or calendar-app. I know there are some things that have slipped through, but the goal is for common to be totally agnostic. The solution here would be to either create generic SearchBar and SearchBarAttrs that Mail/Calendar SearchBar implement, or split LazySearchBar and have a Mail and Calendar version. (I would personally go for the latter.)

Reply:
Good catch, we now have two LazySearchBars that are injected from the entry.